.github/workflows/doctoc.yml

```
@@ -0,0 +1,27 @@
+name: Check generated TOCs
+
+on:
+  pull_request:
+    paths:
+      - "README.md"
+      - "docs/BestPractices.md"
+
+jobs:
+  doctoc:
+    name: Doc TOC Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install doctoc
+        run: npm i -g doctoc
+      - name: Create README copy and diff with doctoc
+        run: cp README.md README.md.tmp &&
+          doctoc --title='## Table of Contents' --github README.md &&
```

```suggestion
          npx doctoc --title='## Table of Contents' --github README.md &&
```

and ditch the install?

---

With multiple calls, is it better to do the install? I don't use NPX much

---

Testing locally it looks like running it with NPM re-validates the cache each execution, so I don't think it makes sense with the multiple calls in this job

---

Fair enough 🙂